### DISCUSSION TITLE
I still didn't receive my benefits developer pack

### DISCUSSION THREAD

**Discussion by luisrca-tech** [community/community#134446] [GitHub Education]
### Hi everyone,

Hi everyone, How long after approval did you receive the benefits, I still haven't received them and it's been 4 days, is this normal?

![Screenshot 2024-08-02 163802](https://github.com/user-attachments/assets/57cd0c54-28f6-4795-a0aa-6f0ee2feca10)

**Comment by davevad93** [comment]
Hi @luisrca-tech ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @luisrca-tech was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
